Portfolio/index.css

```diff
@@ -1 +1,174 @@
-/* Write your code below for PORTFOLIO */
+console.log('Hello World');
+

```

Comment:
The 'body' selector should be preceded by a '.' to indicate that it is a class selector